easycv/models/detection/fcos/fcos_head.py

```diff
@@ -0,0 +1,831 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+
+import numpy as np
+import torch
+import torch.nn as nn
+from mmcv.cnn import ConvModule, Scale
+from mmcv.ops import batched_nms
```

Comment:
use torchvision nms for better compatibility

Reply:
from mmcv description：

 Modified from `torchvision/ops/boxes.py#L39
    <https://github.com/pytorch/vision/blob/
    505cd6957711af790211896d32b40291bea1bc21/torchvision/ops/boxes.py#L39>`_.
    In order to perform NMS independently per class, we add an offset to all
    the boxes. The offset is dependent only on the class idx, and is large
    enough so that boxes from different classes do not overlap.

Reply:
if  `from mmcv.ops import batched_nms` runs successfully we can use nms from mmcv, otherwise we use nms from torchvision

Reply:
done